modules/customizing-the-global-header/proc-customizing-rhdh-global-header.adoc

```diff
@@ -0,0 +1,38 @@
+[id="customizing-your-product-global-header_{context}"]
+= Customizing your {product} global header
+
+To replace the {product} global header with a custom header of choice, disable the default header.
+
+.Prerequisites
+* You have configured your {product-custom-resource-type} instance.
+* You have configured the support URL in the `app-config.yaml` file.
+* You have installed the notification plugin.
+
+.Procedure
+
+* Install your own header as a dynamic plugin as shown in the following example:
```

Comment:
Procedure steps should be a numbered list. See  https://redhat-documentation.github.io/modular-docs/#con-creating-procedure-modules_writing-mod-docs and https://redhat-documentation.github.io/modular-docs/#modular-docs-procedure-examples

```suggestion
. Install your own header as a dynamic plugin as shown in the following example:
```

Reply:
Additionally, specify in the step which file the user is editing.